nautobot_secrets_providers/urls.py

```diff
@@ -3,12 +3,21 @@
 from django.templatetags.static import static
 from django.urls import path
 from django.views.generic import RedirectView
+from nautobot.apps.urls import NautobotUIViewSetRouter
 
-from nautobot_secrets_providers import views
 
-app_name = "nautobot_secrets_providers"
```

Comment:
@gsnider2195 Is this no longer necessary, or is this an issue we need to track?

Reply:
Probably just a bad merge conflict resolution? This is still required.

Reply:
Looks like it's missing from the cookie so we probably do need an issue opened

Reply:
Should this be manually reverted for now?